app/services/katello/pulp/repository/puppet.rb

```diff
@@ -18,8 +18,15 @@ def generate_mirror_importer
           importer_class.new(config.merge(mirror_importer_connection_options))
         end
 
+        def auto_publish?
```

Comment:
can you rename this to be more specific?  puppet_install_auto_publish?  or something like that